### ISSUE TITLE
discord_utilities don't give role

### ISSUE THREAD

**Issue by UsagiHatsuko** [Cruze03/discord-utilities#13]
L 04/19/2021 - 12:38:35: [SM] Blaming: discord_utilities.smx
L 04/19/2021 - 12:38:35: [SM] Call stack trace:
L 04/19/2021 - 12:38:35: [SM]   [0] ThrowError
L 04/19/2021 - 12:38:35: [SM]   [1] Line 75, C:\Users\STARS\Desktop\discord-utilities - Copy\discord-utilities-master\scripting\include\multicolors/colors.inc::C_PrintToChat
L 04/19/2021 - 12:38:35: [SM]   [2] Line 76, C:\Users\STARS\Desktop\discord-utilities - Copy\discord-utilities-master\scripting\include\multicolors.inc::CPrintToChat
L 04/19/2021 - 12:38:35: [SM]   [3] Line 520, discord_utilities/forwards.sp::Command_ViewId

**Comment by Erik-97** [comment]
it is just make sure you gave the right permissions to the bot and/or channel

**Comment by Cruze03** [comment]
Yes and try using latest version of discord utilities and discord api (in my repository)
